src/main/java/dk/kb/netarchivesuite/solrwayback/util/UrlUtils.java

```diff
@@ -9,13 +11,81 @@
 import java.util.regex.Pattern;
 import java.util.stream.Stream;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class UrlUtils {
 
-    public static void main(String[] args){
+    private static final Logger log = LoggerFactory.getLogger(UrlUtils.class);
+    
+    public static void main(String[] args) throws Exception{
 
         //System.out.println(isUrlWithDomain("http://Portal_gfx/KL/farvepakker/topmenu/topmenu_markering_groen_mBo.gif"));
         //System.out.println(getDomainFromWebApiParameters("http://teg-desktop.sb.statsbiblioteket.dk:8080/solrwayback/services/web/20071221033234/http://kl.dk/ncms.aspx?id=11fb172c-dbf2-4c58-bdf7-30f9cdfd4d95&menuid=361285"));
+
+        //Normalisation.setType(NormaliseType.LEGACY);
+        //System.out.println(fixLegacyNormaliseUrlErrorQuery("http://bt.dk/"));
+    
+    }
+    
+
+    /**
+    //THis method will be removed at a later point when Royal Danish Library has re-indexed everything again using warc-indexer 3.2+
+    // For pure domain URL (no paths) such as 'http://abc.dk/' and 'http:///www.abc.dk/' , they are two different url_norm with the legacy normalizer.
+    // So in this case return both options:
+    // For non-legacy:   url_norm:"http://abc.dk/"
+    // For legacy:       url_norm:("http://abc.dk/" OR "http://www.abc.dk/")        
+    */
```

Comment:
The method seems solid and thanks for the JavaDoc. Only request is to replace `//` with `*` in the JavaDoc.

Reply:
Okay, maybe another consideration: The classes `Normalisation`, `NormalisationStandard` and `NormalisationLegacy` already handle URL-normalisation. Why is `fixLegacyNormaliseUrlErrorQuery` in the `UrlUtils` class and not in one of the 3 normalisation-classes?